staging/src/k8s.io/pod-security-admission/test/run.go

```diff
@@ -160,7 +181,7 @@ func (t *testWarningHandler) FlushWarnings() []string {
 // and ensures pod fixtures expected to pass and fail against that level/version work as expected.
 func Run(t *testing.T, opts Options) {
 	warningHandler := &testWarningHandler{}
-
+	const podOSBasedRestrictionEnabledVersion = 25
```

Comment:
can you drop this and use the package var `podOSBasedRestrictionEnabledVersion` which is already declared?